main.js

```diff
@@ -79,13 +86,16 @@ function submitForm() {
   event.preventDefault();
   for (var i=0; i < inputFields.length; i++) {
     if (!getCategory()) {
+      categoryWarning.innerHTML = `<img src="./assets/warning.svg" alt="warning sign" id="warning"> A category is required.`;
```

Comment:
Let's remember to make sure that the message does not disappear once user clicks a category button.